app/models/experimentation/notifications_experiment.rb

```diff
@@ -54,6 +55,14 @@ def self.exclude_bangladesh_blocks(patients)
         .where.not(block_region: {id: excluded_block_ids.presence})
     end
 
+    def self.restrict_indian_states(patients)
```

Comment:
Rename the method to `exclude_non_experiment_indian_states`?
`exclude` seems better than `restrict`

Reply:
wdyt of `include_selected_indian_states`? It applies a whitelist so it's not actually `exclude`ing.

Reply:
What this function is doing is removing non_experiment states from the list.
I'll prefer `filter_non_experiment_indian_states` to `include_experiment_indian_states`. Your call. No more discussion on this.

Reply:
I like `exclude_india_states`. Consistent with the exclusion above (exclude_bangladesh_blocks). Note that it's `India`, not `Indian`, to be consistent 😅

Reply:
>What this function is doing is removing non_experiment states from the list.
There is no notion of `non_experiment` states, only `included` states.

Going with `exclude_india_states`, can't think of a name that captures the whitelist well.

Reply:
Actually, `include_only_allowed_india_states` fits the bill. Picking this.